packages/core/src/tools/create-rpc-client.js

```diff
@@ -108,7 +108,7 @@ const createRpcClient = (event) => {
       try {
         res = await request(req);
 
-        if (res.status > 500) {
+        if (res.status >= 500) {
```

Comment:
HTTP 500 is now a non-standard response code (note this PR is targeting a major release branch as this could be a breaking change)